R/pkg/R/DataFrame.R

```diff
@@ -629,8 +629,9 @@ setMethod("repartition",
 #' @param x A SparkDataFrame
 #' @return A StringRRDD of JSON objects
 #' @family SparkDataFrame functions
-#' @rdname tojson
-#' @noRd
+#' @rdname toJSON
+#' @name toJSON
```

Comment:
wait, why are we changing this from `@noRd`? this is not exported from SparkR and should not be documented.